addons/hw_drivers/iot_handlers/drivers/L10nKeEDISerialDriver.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from collections import namedtuple
+import logging
+import re
+import serial
+import threading
+import time
+import struct
+import json
+from functools import reduce
+from base64 import urlsafe_b64encode, urlsafe_b64decode
+
+from odoo import http
+from odoo.addons.hw_drivers.controllers.proxy import proxy_drivers
+from odoo.addons.hw_drivers.event_manager import event_manager
+from odoo.addons.hw_drivers.iot_handlers.drivers.SerialBaseDriver import SerialDriver, SerialProtocol, serial_connection
+from odoo.addons.hw_drivers.main import iot_devices
+
+_logger = logging.getLogger(__name__)
+
+TremolG03Protocol = SerialProtocol(
+    name='Tremol G03',
+    baudrate=115200,
+    bytesize=serial.EIGHTBITS,
+    stopbits=serial.STOPBITS_ONE,
+    parity=serial.PARITY_NONE,
+    timeout=3,
+    writeTimeout=0.2,
+    measureRegexp=None,
+    statusRegexp=None,
+    commandTerminator=b'',
+    commandDelay=0.2,
+    measureDelay=0.2,
+    newMeasureDelay=0.2,
+    measureCommand=b'',
+    emptyAnswerValid=False,
+)
+
+STX = b'\x02'
+ETX = b'\x0A'
+ACK = b'\x06'
+NACK = b'\x15'
+
+
+class TremolG03Driver(SerialDriver):
+    """Driver for the blackbox fiscal data module."""
+
+    _protocol = TremolG03Protocol
+
+    def __init__(self, identifier, device):
+        super().__init__(identifier, device)
+        self.device_type = 'ke_fiscal_device'
+        self.message_number = 0
+
+    @classmethod
+    def get_default_device(cls):
+        fiscal_devices = list(filter(lambda d: iot_devices[d].device_type == 'ke_fiscal_device', iot_devices))
+        return len(fiscal_devices) and iot_devices[fiscal_devices[0]]
+
+    @classmethod
+    def supported(cls, device):
+        """Checks whether the device, which port info is passed as argument, is supported by the driver.
+
+        :param device: path to the device
+        :type device: str
+        :return: whether the device is supported by the driver
+        :rtype: bool
+        """
+        protocol = cls._protocol
+        try:
+            protocol = cls._protocol
+            with serial_connection(device['identifier'], protocol) as connection:
+                connection.write(b'\x04')
```

Comment:
You can use `connection.write(b'\x09')`  and `response == b'\x40'` it is more specific to your device
check `1.3. SHORT MESSAGES FOR TESTING THE STATUS OF THE CU`